core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java

```diff
@@ -72,8 +76,12 @@ public void handle(final RoutingContext routingContext) {
                 if (slashingProtection.isPresent()) {
                   try {
                     if (maySign(Bytes.fromHexString(identifier), eth2SigningRequestBody)) {
+                      slashingMetrics.incrementSigningsPermitted(
+                          eth2SigningRequestBody.getType().name());
                       respondWithSignature(routingContext, signature);
                     } else {
+                      slashingMetrics.incrementSigningsPrevented(
+                          eth2SigningRequestBody.getType().name());
                       LOG.debug("Signing not allowed due to slashing protection rules failing");
                       routingContext.fail(403);
                     }
```

Comment:
If the slashing protection failed due to some error, say a DB error do want to capture that somewhere too? Though perhaps that should just fall into some httpMetric for failing request

Reply:
We already capture how many requests we receive - regardless of the failure, we didn't sign ... so for now 🤷 I'm happy to catch db failures in this category.... otherwise this needs to be captured WAY down in the slashingdb .. which maybe we should do